draft-ietf-httpbis-header-structure.md

```diff
@@ -777,7 +781,8 @@ NOTE: This algorithm parses both Integers {{integer}} and Floats {{float}}, and
    6. If type is "float" and input_number contains more than 16 characters, fail parsing.
 8. If type is "integer":
    1. Parse input_number as an integer and let output_number be the product of the result and sign.
-   2. If output_number is outside the range defined in {{integer}}, fail parsing.
+   2. If output_number is outside the range for integers defined in {{integer}}, fail parsing.
```

Comment:
The two lines could be one, as follows:
```suggestion
   2. If output_number is outside the range for integers supported by the parser (see {{integer}}), fail parsing.
```

Reply:
I'd rather be explicit.

Reply:
In which case, your two points need to be switched.  As written, this would prohibit supporting a larger range.

Reply:
Belay that.  The comment stands.  There is no way to have a fail parsing on this condition.

Reply:
I don't think it prohibits supporting a larger range directly, but it does beg the question of how you can check that the number falls within the greater range if you can't accurately represent it. Will fix.